.github/workflows/build-and-test.yml

```diff
@@ -4,15 +4,20 @@ on: [push, pull_request]
 
 jobs:
   build-and-test:
-    runs-on: ubuntu-latest
-
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        java: [8, 8.0.192, 11, 11.0.9, 12, 13, 13.0.4, 14.0.2, 15]
```

Comment:
I'd just leave it to the major versions and not the patch releases, also why test versions that are already EOL (OpenJDK 12 /13 / 14 have all passed)... 
I'd just stick to LTS versions (8 and 11)

Reply:
Michael,
Sounds good to avoid patch release, however I’d keep all major versions. I and others use Java/JavaFX 14 & 15! 
Btw, I did test 16ea and it failed for some reason so I took it out.

Reply:
Assuming you agree with me. Did you want me to make the change or will you make the change?

Reply:
I had made the change to remove EOL and patched versions.